target/linux/ath79/generic/base-files/etc/hotplug.d/firmware/11-ath10k-caldata

```diff
@@ -196,6 +196,13 @@ case "$FIRMWARE" in
 			/lib/firmware/ath10k/QCA9888/hw2.0/board.bin
 		rm /lib/firmware/ath10k/QCA9888/hw2.0/board-2.bin
 		;;
+	dlink,covr-p2500-a1)
+		caldata_extract "art" 0x5000 0x2f20
+		ath10k_patch_mac $(mtd_get_mac_ascii art "protest_ath1_mac")
+		ln -sf /lib/firmware/ath10k/pre-cal-pci-0000\:00\:00.0.bin \
```

Comment:
I think this part can also be converted to `pre-calibration` NVMEM, too.

Reply:
Ref: https://git.openwrt.org/openwrt/openwrt.git?p=openwrt%2Fopenwrt.git&a=search&h=HEAD&st=grep&s=pre-calibration

Reply:
Converted `pre-calibration` and `mac-address` to NVMEM. I am not sure whether offset for `mtd_get_mac_ascii art "protest_ath1_mac"` is same `0xb8` on every device
```
00000000  ff ff ff ff ff ff ff ff  ff ff ff ff ff ff ff ff  |................|
*
00000020  63 68 65 63 6b 73 75 6d  3d 58 58 58 58 58 58 58  |checksum=XXXXXXX|
00000030  58 0a 66 61 63 74 6f 72  79 6d 6f 64 65 3d 30 0a  |X.factorymode=0.|
00000040  70 72 6f 74 65 73 74 5f  70 6c 63 5f 6d 61 63 3d  |protest_plc_mac=|
00000050  58 58 3a 58 58 3a 58 58  3a 58 58 3a 58 58 3a 30  |XX:XX:XX:XX:XX:0|
00000060  31 0a 70 72 6f 74 65 73  74 5f 6c 61 6e 5f 6d 61  |1.protest_lan_ma|
00000070  63 3d 58 58 3a 58 58 3a  58 58 3a 58 58 3a 58 58  |c=XX:XX:XX:XX:XX|
00000080  3a 30 32 0a 70 72 6f 74  65 73 74 5f 61 74 68 30  |:02.protest_ath0|
00000090  5f 6d 61 63 3d 58 58 3a  58 58 3a 58 58 3a 58 58  |_mac=XX:XX:XX:XX|
000000a0  3a 58 58 3a 30 33 0a 70  72 6f 74 65 73 74 5f 61  |:XX:03.protest_a|
000000b0  74 68 31 5f 6d 61 63 3d  58 58 3a 58 58 3a 58 58  |th1_mac=XX:XX:XX|
000000c0  3a 58 58 3a 58 58 3a 30  34 0a 70 6c 63 5f 64 65  |:XX:XX:04.plc_de|
000000d0  6b 3d 58 58 58 58 2d 58  58 58 58 2d 58 58 58 58  |k=XXXX-XXXX-XXXX|
000000e0  2d 58 58 58 58 0a 70 6c  63 5f 6e 65 74 77 6f 72  |-XXXX.plc_networ|
000000f0  6b 70 77 64 3d 58 58 58  58 2d 58 58 58 58 2d 58  |kpwd=XXXX-XXXX-X|
00000100  58 58 58 2d 58 58 58 58  0a 73 73 69 64 3d 64 6c  |XXX-XXXX.ssid=dl|
00000110  69 6e 6b 2d 58 58 58 58  0a 77 6c 61 6e 70 61 73  |ink-XXXX.wlanpas|
00000120  73 77 64 3d 68 65 6d 66  6c 58 58 58 58 58 0a 70  |swd=hemflXXXXX.p|
00000130  6c 63 5f 63 6f 75 6e 74  72 79 63 6f 64 65 3d 47  |lc_countrycode=G|
```

Reply:
> Converted `pre-calibration` and `mac-address` to NVMEM. I am not sure whether offset for `mtd_get_mac_ascii art "protest_ath1_mac"` is same `0xb8` on every device

These MAC addresses are generated and written by the same program, so I think they should have the same offset.

Reply:
Changed `mac-address` logic back to `mtd_get_mac_ascii`